src/css/application.css

```diff
@@ -329,14 +329,32 @@ body {
   flex: 1 1 auto;
   display: flex;
   flex-direction: row;
+  height: 100vh;
```

Comment:
@jwang1919 OK, I’m convinced—have you checked on the behavior in Firefox and Safari? This does seem like an area where the browsers’ subtly differing flex behavior might bite us…